src/commands/tree_of.rs

```diff
@@ -88,10 +90,24 @@ pub async fn tree_of(matches: &ArgMatches, repo: Repository, config: &Configurat
                             }
                         )
                     },
-                    &|_, node| format!("label = \"{}\" ", node.1.display_name_version()),
+                    &|_, node| format!("label = \"{}\" ", node.1.clone().display_name_version()),
```

Comment:
That's weird - something must've gone wrong here (since the else branch has no effect on this part)